lib/matplotlib/tests/test_ticker.py

```diff
@@ -238,15 +239,132 @@ def test_set_params(self):
             loc.set_params()
 
 
+class LogitHelper:
+    @staticmethod
+    def isclose_in_logitspace(x, y):
```

Comment:
perhaps make these toplevel (private) helpers (`_is_close_in_logitspace`) or at least don't repeat "logit" in the name? (`_LogitSpace.is_close`, `_LogitSpace.assert_almost_equal`)?  The repetition of "logit" makes lines unduly long below.

Reply:
I rename the class as private. In remove the repetition for logit in the function names.

For testing I am generally less astringent for private/public object. It why I didn't put this helper as private.